### PR TITLE
Assignment 1 CP MP- Ankit Jain

### PR DESCRIPTION
**Pull request by jain2anki**

### CODE REVIEW

**Comment by jain2anki** [comment]
Assignment 1 CP SMP Ankit Jain

File: CP/Assignment1-STL/Ankit_Jain/pair.cpp.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#include<bits/stdc++.h>
+using namespace std;
+bool myfunc(pair<int,int> &a, pair<int,int> &b) // I didn't actually get this concept.
+{
+return a.second<b.second;
```
Comment (Aj163) [inline CP/Assignment1-STL/Ankit_Jain/pair.cpp.txt]: If second is equal check for first

Comment (jain2anki) [reply CP/Assignment1-STL/Ankit_Jain/pair.cpp.txt]: I dont know how to change them to .cpp

Comment (Aj163) [reply CP/Assignment1-STL/Ankit_Jain/pair.cpp.txt]: Make a new file (pair.cpp not pair.cpp.txt) and put your code there, delete these old ones

Comment (jain2anki) [reply CP/Assignment1-STL/Ankit_Jain/pair.cpp.txt]: there is an option for changing the file using online editor. shall I change the file name using it? will it work

File: CP/Assignment1-STL/Ankit_Jain/pair.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#include<bits/stdc++.h>
+using namespace std;
+bool myfunc(pair<int,int> &a, pair<int,int> &b) // I didn't actually get this concept.
+{
+return a.second<b.second;
```
Comment (Aj163) [inline CP/Assignment1-STL/Ankit_Jain/pair.cpp]: If second values are equal check for first values

File: CP/Assignment1-STL/Ankit_Jain/pair.cpp [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#include<bits/stdc++.h>
+using namespace std;
+bool myfunc(pair<int,int> &a, pair<int,int> &b) // I didn't actually get this concept.
```
Comment (Aj163) [inline CP/Assignment1-STL/Ankit_Jain/pair.cpp]: If you didn't get the concept, ask a mentor, no point mentioning it here.

File: CP/Assignment1-STL/Ankit_Jain/pair.cpp (hunk 2 above)
Comment (Aj163) [reply CP/Assignment1-STL/Ankit_Jain/pair.cpp]: Do this change, everything else is fine.

**Pull request merged**